tests/Fixtures/TestBundle/Entity/SecuredDummy.php

```diff
@@ -84,6 +84,17 @@ class SecuredDummy
      */
     private $adminOnlyProperty = '';
 
+    /**
+     * @var string Secret property, only readable/writable by owners
+     *
+     * @ORM\Column
+     * @ApiProperty(
+     *     security="object == null or object.getOwner() == user",
```

Comment:
Is it still useful?

Reply:
As the `object` isn't available in creates/updates, in this scenario you'd need to pass the `security` check if the `object`is `null`, as otherwise it would fail this check (`object.getOwner()` would throw a null pointer exception) and never run `security_post_denormalize`.

Reply:
But if there is already a `securityPostDenormalize`, why a `security` is needed?

Reply:
Found the answer myself: it's because it is used for the `get` operation too.